src/components/App/index.js

```diff
@@ -33,11 +36,24 @@ class App extends Component {
       </section>
       <section className="Registration">
 
+      </section>
+      <section className="App-Ideas-Section">
+        <div className="App-Ideas-Section-Container">
+          {
+            (ideas || []).map((idea) => {
```

Comment:
what will it do when the array is empty? just won't show anything?

Reply:
yes, just will not iterate. 
I do it because if I won't, and there are no items, it will throw an error and break everything.